src/status_im/chat/models/input.cljs

```diff
@@ -126,6 +126,15 @@
                                             :image-path   (string/replace image-path #"file://" "")
                                             :text         "Update to latest version to see a nice image here!"})))))
 
+(fx/defn send-audio-message
+  [cofx audio-path duration current-chat-id]
+  (when-not (string/blank? audio-path)
+    (chat.message/send-message cofx {:chat-id           current-chat-id
+                                     :content-type      constants/content-type-audio
+                                     :audio-path        audio-path
+                                     :audio-duration-ms duration
+                                     :text              "Update to latest version to listen to an audio here!"})))
```

Comment:
`to an audio message` maybe?

Reply:
and moreover those messages should be all in i18n :-)